index.html

```diff
@@ -10,7 +10,25 @@
 </head>
 
 <body>
-    <h1>Hello</h1> 
+    <main id="main">
+        <div>
```

Comment:
Hi, @pradmon!! 👋
- Here you have made a good suggestion politely using reasoning. 👍 

- [ ] Kindly use emoticon (😄) so that your review looks human-friendly.

Reply:
Good afternoon, the changes were made according to the request.
thanks 👍